leda/components/AutoComplete/types.ts

```diff
@@ -96,69 +101,67 @@ export interface BlurEvent extends React.FocusEvent<HTMLInputElement> {
+  /** Search mode, smart is default, looks for one or several words regardless of their order, can be slow if data has thousands of elements or more */
   filterRule?: 'smart' | 'startsWith' | 'includes',
-  /** Футер под значениями в выпадающем списке */
-  footerRender?: () => React.ReactNode,
-  /** Ключ для группировки */
+  /** How to group suggestions */
   groupBy?: (option: T) => string | undefined,
-  /** Хедер над значениями в выпадающем списке */
-  headerRender?: () => React.ReactNode,
-  /** Отображение кнопки для очистки значения в инпуте */
+  /** Whether or not to show a clear button inside the input element. Default is false */
   hasClearButton?: boolean,
-  /** Вместо выпадающего списка в момент загрузки будет отображаться лоадер - полезно при подгрузке данных для списка с сервера */
+  /** Display a loading icon inside the dropdown */
   isLoading?: boolean,
-  /** Рендерить компонент с открытым выпадающим списком */
+  /** Control the dropdown state */
   isOpen?: boolean,
-  /** Кастомизация внешнего вида элемента выпадающего списка. */
+  /** Siggestion item customizator */
   itemRender?: SuggestionListProps['itemRender'],
-  /** Кастомизация внешнего вида выпадающего списка. */
+  /** Input field customizator */
   inputRender?: CustomRender<AutoCompleteProps, AutoCompleteState, InputElementProps>,
-  /** Обязательное поле или нет */
+  /** Required or not */
   isRequired?: boolean,
-  /** Кастомизация внешнего вида выпадающего списка. */
+  /** Dropdown list customizator */
   listRender?: CustomRender<SuggestionListProps, {}, UlProps>,
-  /** Количество символов в инпуте после которых начать показывать выпадающий список */
+  /** The minimal number of symbols that triggers the dropdown opening */
   minSearchLength?: number,
-  /** Имя поля ввода */
+  /** Component name */
   name?: string,
   /** This will be shown when no suggestions are found */
   noSuggestionsText?: React.ReactNode,
-  /** Принимает JSX */
+  /** No suggetions block customizator */
   noSuggestionsRender?: SuggestionListProps['noSuggestionsRender'],
-  /** Обработчик события потери фокуса */
+  /** Blur handler */
   onBlur?: (event: BlurEvent) => void,
-  /** Обработчик события изменения значения в инпуте */
+  /** Change handler */
   onChange: (event: ChangeEvent<T>) => void,
-  /** Обработчик события фокуса */
+  /** Focus handler */
   onFocus?: (event: FocusEvent) => void,
-  /** Плейсхолдер */
+  /** Placeholder */
   placeholder?: string,
-  /** При потере фокуса проверяет введенное значение на соответствие значениям в data и подставляет последнее корректное (есть в data) или пустое значение */
+  /** If true it puts the last correct (present in data) value into the input field or clears it. It is triggered by Blur event */
   shouldCorrectValue?: boolean,
-  /** Показывать все элементы из data, не фильтруя */
+  /** Show all data elements regardless of what is in the input field */
```

Comment:
Or just disable fusion filtration?)

Reply:
the main idea here is to show all suggestions